catalog/task/slack-webhook-notification/0.1/samples/slack-webhook-notification-TaskRun.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: slack-webhook-notification-run-empty-params
+spec:
+  params:
+    - name: message
+      value: ""
+    - name: extraDataJsonPath
```

Comment:
you can omit the parameters with a default in the sample

Reply:
fixed